app/views/admins/index.html.erb

```diff
@@ -1,9 +1,26 @@
-<%=@user.first_name%>
+Welcome <%=@user.first_name %> | <%= link_to "Facilities", show_facilities_admins_path %>
```

Comment:
If you are displaying the logged in username you can just use `current_user.first_name`

Reply:
Changed to current_user.first_name and also deleted @user in the controller.